src/nvim/ascii.h

```diff
@@ -87,4 +90,45 @@
 # define PATHSEPSTR     "/"
 #endif
 
+static inline bool ascii_iswhite(int c) FUNC_ATTR_ALWAYS_INLINE FUNC_ATTR_CONST;
```

Comment:
very small nitpick, but given this matches `isblank's` behavior (besides respecting locale), maybe this could be `ascii_isblank`? I say this because all the other functions just have `ascii_` prepended to them, but this one has a different name as well.